build/rust/lib.rs

```diff
@@ -3,3 +3,6 @@ pub use adblock;
 pub use challenge_bypass_ristretto_ffi;
 
 pub use speedreader_ffi;
+
+#[cfg(target_os = "windows")]
```

Comment:
same as https://github.com/brave/brave-core/pull/7365/files#r538592327

Reply:
Not using rust anymore.